crates/nargo/tests/test_data/testing/src/main.nr

```diff
@@ -0,0 +1,5 @@
+fn main(x : u32, y : pub u32) {
```

Comment:
@vezenovm are we supposed to move these to nargo_cli?

Reply:
We can probably remove this test. Or move this to `nargo_cli` to have a pure logic constraint test. We also test logic constraints in `8_integration` just a lot more is going on in that test.

Reply:
I moved it to an `xor` test under `nargo_cli` lmk what you think